### DISCUSSION TITLE
Other Operators

### DISCUSSION THREAD

**Discussion by darren2005** [LiveHelperChat/livehelperchat#1919] [General]
Hi, is there some way of disabling the view of other operators? e.g. I have a number of different domains with different projects and when clicking on the mobile app, I can see all operators across my different projects. I don't want this as each project is different.

**Comment by remdex** [comment]
I already wrote...
https://doc.livehelperchat.com/docs/dashboard#online-operators
https://doc.livehelperchat.com/docs/chat/multiple-clients

**Reply by darren2005** [reply]
Thanks again! Maybe the issue is that when I try config the widget, it does not show the departments I've just created - see image - there should be another x2 depts in that list? 

https://www.dropbox.com/s/w0pf1u6o1ud0wd8/Screenshot%202023-03-05%2013.24.02.png?dl=0

**Reply by remdex** [reply]
You have to assign department to yourself...

**Reply by darren2005** [reply]
Ok, BINGO! I think I'm on the right track... One more question,  now the user on the mobile app cannot change his/her online status? what am I missing?

**Reply by remdex** [reply]
They can change that on mobile, so dig deeper...

**Reply by darren2005** [reply]
:( Ok, at the moment only admins can change status.. I'll take a look

**Comment by remdex** [comment]
Look into permissions…

On 2023-03-05, Sun at 16:36, darren2005 ***@***.***> wrote:

> :( Ok, at the moment only admins can change status.. I'll take a look
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/LiveHelperChat/discussions/1919#discussioncomment-5207397>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIXZZOLYWB5NL3QW7AQUTDW2SQFJANCNFSM6AAAAAAVQFCSJQ>
> .
> You are receiving this because you commented.Message ID:
> <LiveHelperChat/livehelperchat/repo-discussions/1919/comments/5207397@
> github.com>
>
-- 
Sincerely,
Remigijus Kiminas

**Reply by darren2005** [reply]
Thanks for your help on this! Much appreciated.. I'll take a look into permissions.

**Comment by darren2005** [comment]
Hi, 
   
  
      
   
  
    Sorry to bother you again. 
   
  
      
   
  
    I'm struggling to change user permissions and when I do, the permissions remain the same. 
   
  
      
   
  
    How do I change the below? How do I untick the boxes? 
   
  
      
   
   
    
   
  
      
   
  
    thanks, 
   
  
      
   
  
    Daz 
   
   
   
     On 05/03/2023 14:36 GMT Remigijus Kiminas ***@***.***> wrote: 
    
   
       
    
   
       
    
     Look into permissions… 
    
   On 2023-03-05, Sun at 16:36, darren2005 ***@***.***> wrote: 
    
   > :( Ok, at the moment only admins can change status.. I'll take a look 
   > 
   > — 
   > Reply to this email directly, view it on GitHub 
   > <https://github.com/orgs/LiveHelperChat/discussions/1919#discussioncomment-5207397>, 
   > or unsubscribe 
   > <https://github.com/notifications/unsubscribe-auth/AAIXZZOLYWB5NL3QW7AQUTDW2SQFJANCNFSM6AAAAAAVQFCSJQ> 
   > . 
   > You are receiving this because you commented.Message ID: 
   > <LiveHelperChat/livehelperchat/repo-discussions/1919/comments/5207397@ 
   > github.com> 
   > 
   -- 
   Sincerely, 
   Remigijus Kiminas
